code/game/area/areas/ruins/jungle.dm

```diff
@@ -96,3 +96,33 @@
 	name = "cargo dump"
 	icon_state = "dk_yellow"
 
+// Syndicate Shitfort
```

Comment:
This is quite an extreme amount of areas that could easily be replaced with only 1 unique area and some basic general areas.

Reply:
I gave Jerry all the shutters and stuff. The rest is the generic syndicate fort or away mission now.